lib/Twig/Assets.php

```diff
@@ -57,8 +57,9 @@ function generateJavascript() {
     $output = array();
 
     foreach($scripts as $script) {
+      $dir = substr($script, 0, 4) == 'lib/' ? 'js' : 'dist/js';
       $output[] = sprintf(
-        '<script type="text/javascript" src="%s/js/%s"></script>',
+        '<script type="text/javascript" src="%s/'.$dir.'/%s"></script>',
```

Comment:
I thought our coding standards require separating `.` by two spaces. But if you are using `sprintf` anyway why do you need to combine two different styles of building string? But I think the best approach here would be to build the whole path to the script file at the beginning and then use it here with a single `%s`.